eva_cttv_pipeline/trait_mapping/main.py

```diff
@@ -13,9 +14,7 @@
 logger = logging.getLogger(__package__)
 
 # These ambiguous trait names cannot be resolved to a specific disease and must not be output
-# TODO: Also use this in the future refactor of the quality control system (see issue #114)
-NONSPECIFIC_TRAITS = {'disease', 'not provided', 'not specified', 'reclassified - variant of unknown significance',
-                      'see cases', 'variant of unknown significance'}
+NONSPECIFIC_TRAITS = ClinVarTrait.NONSPECIFIC_TRAITS
```

Comment:
I would prefer just using `ClinVarTrait.NONSPECIFIC_TRAITS` throughout rather than renaming.

Reply:
Good point, thank you, addressed in https://github.com/EBIvariation/eva-opentargets/pull/214/commits/fc34217bba47de3a7c15c0beccbffb693cde93fb